contracts/AMM/SafeBox.sol

```diff
@@ -0,0 +1,17 @@
+pragma solidity ^0.5.16;
```

Comment:
can be 0.8.0?

Reply:
have to deprecate all synthetix dependencies first